health-sdk/sdk/src/main/res/drawable/ghs_bordered_background.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--TODO to be merged with same drawable from branch IPC 160-->
+<shape android:shape="rectangle" xmlns:android="http://schemas.android.com/apk/res/android">
+    <corners android:radius="@dimen/ghs_small"/>
+    <stroke android:color="@color/ghs_payment_input_background_color" android:width="1dp"/>
+    <solid android:color="@color/ghs_payment_input_border_color"/>
```

Comment:
[Request] We should avoid using colors meant for a certain role in a different place. Can you please create a dedicated color for the background, maybe called `ghs_bordered_background_color`?

Reply:
Changed to suggested naming.